diff --git a/accounts/kernel/createKernelAccount.ts b/accounts/kernel/createKernelAccount.ts
index 16dfce2e3d10037e202e344ae2de943261a28d84..f7eada6aa689cf359cd83fd37b5e6d68e7b7fa61 100644
--- a/accounts/kernel/createKernelAccount.ts
+++ b/accounts/kernel/createKernelAccount.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { satisfies } from "semver"
 import {
     type Address,
diff --git a/index.ts b/index.ts
index b50da7749ca45ff164c55f930bfd8c5dda9d8a66..14ce85513981f32f54f3997ecb8d82878439c97b 100644
--- a/index.ts
+++ b/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 export {
     createKernelAccount,
     createKernelAccountV0_2,
