diff --git a/accounts/kernel/utils/signTypedData.ts b/accounts/kernel/utils/signTypedData.ts
index 3eec6345d413eebc5105946c2c9d6576f8eba697..a432e4685a0c90f837d2b5c2b36f3532ce603111 100644
--- a/accounts/kernel/utils/signTypedData.ts
+++ b/accounts/kernel/utils/signTypedData.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import {
     type LocalAccount,
     type SignTypedDataReturnType,
diff --git a/accounts/nexus/toNexusSmartAccount.ts b/accounts/nexus/toNexusSmartAccount.ts
index 213a1242d1baa2745a8c253f6dd42a4f1daf3c49..44dafafd3c06c6425cdfe46306c0aba7e2a57e93 100644
--- a/accounts/nexus/toNexusSmartAccount.ts
+++ b/accounts/nexus/toNexusSmartAccount.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type {
     Account,
     Assign,
diff --git a/index.ts b/index.ts
index 2d108e91d2f3ccb7f7af4e16edbf31b1e4a9e2aa..99eabbffb0912269ccbfaa4976839e3f21256ae9 100644
--- a/index.ts
+++ b/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 export * from "./utils/index.js"
 export * from "./errors/index.js"
 export * from "./clients/index.js"
